test/features/deploy_kogito_build_vs_code.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Deploy Kogito Build - after assets processed by VS code
+  
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed
+    And Project kie-asset-library is cloned
+    And Project kie-asset-library is built by maven with configuration:
+      | profile | dmn                                    |
+      | option  | -Dactive.structures=quarkus            |
+      | option  | -Dactive.config.sets=scesim            |
+      | native  | disabled                               |
+    Then Project "<project>" is generated in temporary folder
+
+
+  Scenario Outline: Build <runtime> binary build with native <native> using KogitoBuild
+    Given Project "<project>" assets are re-marshalled by VS Code
+    And Project "<project>" is built from temporary folder by maven
```

Comment:
Thinking about how to properly describe the action done in here.
The text itself doesn't provide much context, there is no hint of kie-asset-library. 
Maybe 'Project "<project>" in kie-asset-library is built by maven'?

Reply:
With similar syntax in step above.

Reply:
and below too :)

Reply:
yes, thank you, will update the steps of the feature file, your suggestions sounds good to me